spec/support/concerns/organisations.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.shared_examples "it can have organisations" do |schema|
+  before do
+    @example_doc = GovukSchemas::Example.find(schema, example_name: schema)
+    @example_doc["details"]["emphasised_organisations"] = %w[11234500]
```

Comment:
Do we want to hardcode an example here, when I think we want to test the data in the schema that's passed into it?